src/doc/book/primitive-types.md

```diff
@@ -171,12 +171,10 @@ and a length.
 ## Slicing syntax
 
 You can use a combo of `&` and `[]` to create a slice from various things. The
-`&` indicates that slices are similar to [references], which we will cover in
+`&` indicates that slices are similar to [references][references], which we will cover in
```

Comment:
This should actually be identical in Markdown, and I prefer this style. Can you revert these two changes, and just leave the deletion on 178? Thanks!